website/source/docs/drivers/docker.html.md

```diff
@@ -719,6 +726,16 @@ plugin "docker" {
     * `container` - Defaults to `true`. This option can be used to disable Nomad
       from removing a container when the task exits. Under a name conflict,
       Nomad may still remove the dead container.
+    * `dangling_containers` stanza for controlling dangling container detection
+      and cleanup:
+      * `enabled` - Defaults to `true`). Enables dangling container handling
+      * `dry_run` - Defaults to `false`. Enables a mode where nomad logs
+        potential dangling containers without killing them.
+      * `period` - Defaults to `"5m"`.  A time duration that controls interval
+        between Nomad scans for dangling containers.
+      * `creation_grace` - Defaults to `"5m"`.  A time duration that controls
```

Comment:
Ugh, I don't know about my suggestion, but I'd love to document _why_ an operator might use this config. I think the answer is "probably never" so maybe keeping the documentation descriptive instead of prescriptive is fine.

Up to you; not a blocker.

```md
      * `creation_grace` - Defaults to `"5m"`. Grace period after a container is created during which the GC ignores it.
        Only used to prevent the GC from removing newly created containers before they are registered with the GC.
        Should not need adjusting higher but may be adjusted lower to GC more aggressively.
```